tchannel/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from .tchannel import TChannel  # noqa
+from .thrift import from_thrift_module  # noqa
```

Comment:
I mean it doesn't say what's being built. "from_thrift_module" could be building a client, a request builder, a request handler, or anything else. The module name doesn't add any information on what's being build either.

    donut_service = thrift_request_builder(..)

Is equally readable, but it makes it clear that the new `donut_service` object builds requests for donut service. I would even say this makes sense:

    donut_service = ThriftRequestBuilder(...)

(Or Maker instead of Builder.)

My thought here is that there is more than one uses case for doing things dynamically given the module for a single Thrift service.